imqsauth/http.go

```diff
@@ -373,6 +375,55 @@ func httpHandlerCreateUser(central *ImqsCentral, w http.ResponseWriter, r *httpR
 	}
 }
 
+func httpHandlerCreateGroup(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
+	groupname := strings.TrimSpace(r.http.URL.Query().Get("groupname"))
+	if groupname == "" {
+		authaus.HttpSendTxt(w, http.StatusNotAcceptable, "Group name may not be blank.")
+		return
+	}
+
+	if _, error := authaus.LoadOrCreateGroup(central.Central, groupname, true); error == nil {
```

Comment:
Err variables in Go are almost always named 'err'. 'error' is the name of the error type, so it's harder to read